components/footer/aboutInfo/Benefits.tsx

```diff
@@ -0,0 +1,79 @@
+import { FC } from 'react';
+
+import { Box, Grid, Typography } from '@mui/material';
+
+export const Benefits: FC = () => {
+  return (
+    <Grid
+      item
+      marginY="80px"
+      display="flex"
+      width="80%"
+      justifyContent="center"
+      alignItems="center"
+      flexDirection="column"
+    >
+      <Typography variant="h4" textAlign='center'>Nuestros Compromisos</Typography>
```

Comment:
it is recommended to follow the logical and hierarchical order from h1 to h6 in descending order of importance.